client/FileUpload.js

```diff
@@ -54,6 +54,28 @@ class FileUpload extends Component {
             }
             this.setState({fileDoc: fileDoc});
         };
+
+        this.handleLinkBLur = event => {
+            if (this.props.fileType === 'Video') {
+                let fileDoc = this.state.fileDoc;
+                let url = event.target.value;
+                if (!url.includes('youtu.be/')) { // Not a sharable link
```

Comment:
`.startsWith(...)`

Reply:
to use .startsWith() we'd have to account for multiple cases: youtu.be, https://youtu.be, and possibly more depending on where the user got the link. Probably best to stick with includes